plugins/PagePerformance/Visualizations/PerformanceColumns.php

```diff
@@ -26,6 +26,7 @@ class PerformanceColumns extends HtmlTable
 
     public function beforeRender()
     {
+        $this->config->disable_row_evolution = true;
         parent::beforeRender();
```

Comment:
I guess it's better to reverse the order to ensure that the parent method won't change the value again
```suggestion
        parent::beforeRender();
        $this->config->disable_row_evolution = true;
```